xrpl/docs/link-token.md

```diff
@@ -0,0 +1,179 @@
+# XRPL ITS Link Token Guide
+
+## Background
+
+This release provides a generic template for performing custom token linking from an EVM chain to XRPL using the Axelar Interchain Token Service (ITS). 
+
+## Pre-Deployment Setup
+
+Create an `.env` config. Use `all` for `CHAINS` to run the cmd for every EVM chain, or set a specific chain.
+
+```yaml
+PRIVATE_KEY=xyz
+ENV=xyz
+CHAINS=xyz
+```
+**_NOTE:_**
+The private key variable is permanently associated with the custom token registration and linking. The key must be secured since
+it must be reused for future linking of the same token to other chains. The private key is accessed during `evm/*.js` script operations
+
+### 1. Token Symbol Pre-Calculation
+
+Before starting the deployment, you need to generate the XRPL Currency Code for your token symbol.
+
+```bash
+ts-node xrpl/encode-token.js <TOKEN_SYMBOL>
+```
+
+### 2. Environment Variables Setup
+
+| Network              | `XRPL_ISSUER` | `XRPL_GATEWAY` |
+| -------------------- | ------------- | ---------------|
+| **Devnet-amplifier** | `rGAbJZEzU6WaYv5y1LfyN7LBBcQJ3TxsKC`            | `axelar1fqy77ptzsspmewy547dappss2j7scy9asju8qqjts67r2tl4k5cqg6zfxa`         |
+| **Stagenet**         | `rpjRVZbKbb5UmSEcgp6C6U4GzQQF4mEpbR`            | `axelar1vr89h7je4zw4hxhdt7aycqq6end3kf9wvj4unv63s8pcenrrutrs88kn57`         |
+| **Testnet**          | `rNrjh1KGZk2jBR3wPfAQnoidtFFYQKbQn2`            | `axelar18qltw4382s5qz0rgzfxz67mr83smk580hewlkfd33l5tmcdp8unqw35glh`         |
+| **Mainnet**          | `rfmS3zqrQrka8wVyhXifEeyTwe8AMz2Yhw`          | `axelar15dsw0qqnvumnsukjtwt040wnelwrglgslqcqsa7d62f2neuv7slq7rq7zd`         |
+
+
+| Network              | `AXELARNET_GATEWAY` | `ITS_HUB`   |
+| -------------------- | ------------------- | ----------- |
+| **Devnet-amplifier** |  `axelar1wvms3cy5hxrgl7uxhkz7yth4qzqum6aaccwkmvafq8z0mgdfxr8qrnvw0k`  |`axelar157hl7gpuknjmhtac2qnphuazv2yerfagva7lsu9vuj2pgn32z22qa26dk4`              |
+| **Stagenet**         | `axelar1s9amtxejrdlsunwdf0cclhjtezdp6wmurxxnh45gfvtpa2jrsusqv934n6`  | `axelar1ph8qufmsh556e40uk0ceaufc06nwhnw0ksgdqqk6ldszxchh8llq8x52dk`            |
+| **Testnet**          |  `axelar1kq687tszm67hr5ws5pqhtchc8uatxur8r4rm4xgclyghetthtlzs9pnqfl` | `axelar1aqcj54lzz0rk22gvqgcn8fr5tx4rzwdv5wv5j9dmnacgefvd7wzsy2j2mr`              |
+| **Mainnet**          | `axelar18vsne7lns36uvm8gv2cv5jl2lghts0xm7dvzpqzn70dl56gk9hvsgu9sqg`  | `axelar1aqcj54lzz0rk22gvqgcn8fr5tx4rzwdv5wv5j9dmnacgefvd7wzsy2j2mr`       |
+
+Set the following environment variables before running the deployment commands:
+
+```bash
+# Token Details
+XRPL_CURRENCY_CODE= #Generated currency from Token Symbol Pre-Calculation 
+TOKEN_ADDRESS= # Token contract address on the native source chain 
+
+# Deployment Parameters
+SALT= # Random Salt
+TOKEN_MANAGER_TYPE= # Numerical value corresponding to token model (MintBurn, LockUnlock, etc)
```

Comment:
user doesn't know what the value should be. Add this to a table with the two values and explanation on what should be used